doc/input-output-reference/src/hvac-template-objects/group-hvac-templates.tex

```diff
@@ -667,17 +667,19 @@ \subsubsection{Inputs}\label{inputs-3-018}
 
 \paragraph{Field: Cooling Coil Type}\label{field-cooling-coil-type}
 
-There are two coil types available:
+There are three coil types available:
 
 \begin{itemize}
 \item
-  ChilledWater
+  \emph{ChilledWater} uses the Coil:Cooling:Water object with the DetailedAnalysis option. This is the preferred coil model for most applications.
 \item
-  ChilledWaterDetailedFlatModel
+   \emph{ChilledWaterDetailedFlatModel} uses the Coil:Cooling:Water:DetailedGeometry object. This option is available for special applications.
+\item
+   \emph{HeatExchangerAssistedChilledWater} uses CoilSystem:Cooling:Water:HeatExchangerAssisted with Coil:Cooling:Water for enhanced dehumidification.
```

Comment:
Random note here: LaTeX offers a `description` environment which is a nice fit for this sort of keyword-description list.  No worries here, just feel free to have it as a possibility for later.